store/tikv/2pc_test.go

```diff
@@ -668,6 +668,9 @@ func (s *testCommitterSuite) TestElapsedTTL(c *C) {
 // TestAcquireFalseTimeoutLock tests acquiring a key which is a secondary key of another transaction.
 // The lock's own TTL is expired but the primary key is still alive due to heartbeats.
 func (s *testCommitterSuite) TestAcquireFalseTimeoutLock(c *C) {
+	atomic.StoreUint64(&ManagedLockTTL, 1000)       // 1s
```

Comment:
Will it affect other cases?

Reply:
`testCommitterSuite` is a `SerialSuites` suite, and `c.Parallel()` is not used in the test case method, I think it's ok

Reply:
Got